src/models.ts

```diff
@@ -153,3 +153,5 @@ export const defaultSelectedFacets: SelectedFacets = {
   collection: {},
   year: {},
 };
+
+export type EmptyPlaceholderType = 'no-search-term' | 'no-search-result' | '';
```

Comment:
Suggestion: This type does not belong in the model: component data definitions.
It's tightly coupled to the `Placeholder` "component" of `collection-browser.ts` 

First, it should be removed from the model, which has gotten clogged with things that don't belong in it.
Next: and very highly suggested: we want to keep each component as **_simple as possible_**. Please think about "Placeholder" **_and_** its logic as a separate component.

Reply:
```
<empty-placeholder
  .placeholderType=${this.setPlaceholder()}
><\empty-placeholder>
```
seems so much simpler to me

Reply:
Even better: use declarative style removing **_all_** logic from `collection-browser`

```
<empty-placeholder
  .baseQuery=${this.baseQuery}
  .searchResultsLoading=${this.searchResultsLoading}
  .totalResults=${this.totalResults} 
><\empty-placeholder>

Reply:
blocker: Please make this a component.

In discussion with Brenton and the team in our offshoot meeting, I've decided to make this an architectural blocker.